src/ManagedDripsHub.sol

```diff
@@ -78,6 +78,35 @@ abstract contract ManagedDripsHub is DripsHub, UUPSUpgradeable {
         return super.receiveDrips(user, assetId, maxCycles);
     }
 
+    /// @notice Splits user's received but not split yet funds among receivers.
+    /// @param user The user
+    /// @param assetId The used asset ID
+    /// @param currReceivers The list of the user's current splits receivers.
+    /// @return collectableAmt The amount made collectable for the user
+    /// on top of what was collectable before.
+    /// @return splitAmt The amount split to the user's splits receivers
+    function split(
```

Comment:
Okay, this function will stay public because we only increase the `balance.split` and not force a erc20 transfer here

Reply:
Correct!